ee/control/client_http.go

```diff
@@ -106,21 +108,11 @@ func (c *HTTPClient) GetConfig(ctx context.Context) (io.Reader, error) {
 	configReq.Header.Set(HeaderKey, string(key1))
 	configReq.Header.Set(HeaderSignature, sig1)
 
-	// Calculate second signature if available
-	hardwareKeys := agent.HardwareKeys()
-
-	// hardware signing is not implemented for darwin
-	if runtime.GOOS != "darwin" && hardwareKeys.Public() != nil {
```

Comment:
worth noting that this has been this will enable hardware key signing by control server client for darwin